### ISSUE TITLE
Pre-trained models

### ISSUE THREAD

**Issue by GeryAdhane** [zhenxingjian/Partial_Distance_Correlation#4]
Really, incredible work. I really enjoy reading your work and you deserve to win best paper. Two questions: 
1. can we use this DC for calculating the feature distance between two pretrained models and quantify which model has better feature representation (E.g. During knowledge distillation)? E.g. Can we generate the Partial DC in Table 2 of your paper for an already trained network? 

2. Can we use the DC to compare and quantify the similarity of features between two pre-trained models layer by layer?

thank you.

**Comment by zhenxingjian** [comment]
Hi,

Thanks!

1. Actually table 2 in the paper is using the pre-trained network from timm. So I believe this is absolutely doable. For example, in our setting, we are calculating the partial distance correlation of (X|Y, GT) where X is one network, Y is another network (both pre-trained), and the GT is the information from another direction, in our case, we use the BERT embedding of the ground truth labels as the GT here.
2. This is also doable. We show such kind of result in Fig. 3. If we want to summarize the comparison, there should be some extra effort, like either use the average version of the DC between all layers or other statistics; or we can even "concatenate" the features to do the comparison; or in the better way, if we take into the consideration of the structure of the networks, we can use some other "distances" to encode the curvature.

--
Xingjian
